### PR TITLE
A simple script that prints sum of items in a list 

### PR DESCRIPTION
**Pull request by Nagasuruthika**
Issue #91 
Simple Python Script to count items in a list. Prints sum of elements in a list.

### CODE REVIEW

**Comment by Gaurav451** [comment]
I would like to work on this issue

**Review by Gaurav451 — CHANGES_REQUESTED** [review]
In line 5,we can say s=sum(item)

**Comment by Nagasuruthika** [comment]
@Gaurav451 made few changes in code, do check. **Sum(item) produced a type error**.

File: Basic-Scripts/sumofitems_list.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+my_list = [1,2,3,4,5,6,7,8,9,10]
```
Comment (SKAUL05) [inline Basic-Scripts/sumofitems_list.py]: Take input from user

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

**Comment by SKAUL05** [comment]
change file name to `sum_of_items_list`

**Comment by Nagasuruthika** [comment]
@SKAUL05 changes made, do review.

File: Basic-Scripts/sum_of_items_list.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  for item in user_list:
+    answer += item #stores the sum of elements
+  print("Sum of elements in the list:",answer)
+
+
+
+if __name__ == "__main__":
+  user_list = []   
+# number of elemetns as input 
+  n = int(input("Enter number of elements:"))
```
Comment (SKAUL05) [inline Basic-Scripts/sum_of_items_list.py]: num instead of n

File: Basic-Scripts/sum_of_items_list.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+
+if __name__ == "__main__":
+  user_list = []   
+# number of elemetns as input 
+  n = int(input("Enter number of elements:")) 
+
+# iterating till the range 
+  print("Enter the elements:")
+  for i in range(0, n):
```
Comment (SKAUL05) [inline Basic-Scripts/sum_of_items_list.py]: index instead of i

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
Everything else LGTM:)

**Comment by Nagasuruthika** [comment]
@SKAUL05 all done :)

**Review by SKAUL05 — APPROVED** [review]
Good to go

**Pull request merged**